src/client/trackers/realUserMonitoringForPerformance.js

```diff
@@ -52,32 +50,33 @@ export const realUserMonitoringForPerformance = ({ sampleRate } = {}) => {
 	});
 
 	/**
-	 * analyticsTracker()
+	 * recordMetric()
 	 *
 	 * This function is called every time one of the performance events occurs.
 	 * The "final" event should be `firstInputDelay`, which is triggered by any "input" event (most likely to be a click.)
 	 * Once all the metrics are present, it fires a broadcast() to the Spoor API.
 	 */
 	let hasAlreadyBroadcast = false;
 
-	const analyticsTracker = (({ metricName, data}) => {
+	const recordMetric = ((metric) => {
 		if (hasAlreadyBroadcast) return;
 
+		// @see https://github.com/GoogleChrome/web-vitals#metric
+		// for available properties of `metric`
+		const metricName = metric.name.toLowerCase();
+		const data = metric.value;
+
 		if (metricName === 'fid') {
 			context.firstInputDelay = Math.round(data);
 		} else if (metricName === 'lcp') {
 			//This fires twice, we are collecting the 'first firing' - see PR notes for more context https://github.com/Financial-Times/n-tracking/pull/86
 			context.largestContentfulPaint = Math.round(data);
 		} else if (metricName === 'ttfb') {
 			context.timeToFirstByte = Math.round(data);
-		} else if (metricName === 'fp') {
-			context.firstPaint = Math.round(data);
 		} else if (metricName === 'fcp'){
 			context.firstContentfulPaint = Math.round(data);
 		} else if (metricName === 'cls') {
 			context.cumulativeLayoutShift = data;
-		} else if (metricName === 'tbt') {
-			context.totalBlockingTime = Math.round(data);
```

Comment:
**question:** I am guessing these removed metrics are one we care about less and we can happily do without?